apps/aws-notebook/samples/ecs/fargate/vpn/create_app.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${AWS_ACCESS_KEY_ID}" ]] && echo "AWS_ACCESS_KEY_ID is nil" 1>&2 && exit 1
+
+[[ -z "${AWS_SECRET_ACCESS_KEY}" ]] && echo "AWS_SECRET_ACCESS_KEY is nil" 1>&2 && exit 1
+[[ -z "${AWS_SESSION_TOKEN}" ]] && echo "AWS_SESSION_TOKEN is nil" 1>&2 && exit 1
+[[ -z "${AWS_REGION}" ]] && echo "AWS_REGION is nil" 1>&2 && exit 1
+[[ -z "${TASK_IMAGE}" ]] && echo "TASK_IMAGE is nil, please provide the url of notebook image,you could build it from this dockerfile:https://github.com/crafting-demo/solutions/blob/master/apps/aws-notebook/image/Dockerfile and push it to your aws container registry" 1>&2 && exit 1
+[[ -z "${CRAFTING_ORG}" ]] && echo "CRAFTING_ORG is nil, please provide crafting orignization name" 1>&2 && exit 1
```

Comment:
Use `SANDBOX_ORG`:

```sh
[[ -n "$SANDBOX_ORG" ]] || ...
```

BTW, with `set -e`, you can't use `[[ ... ]] && ...`, because if the condition doesn't satisfy, the command will fail immediately without running the part after `&&`, just like condition optimization in other interpreter or compilers. Always use `[[ ... ]] || ...`

Reply:
Also, it will be helpful to define a `fatal` function which takes care of writing message to STDERR rather than STDOUT:

```sh
function fatal() {
  echo "$@" >&2
  exit 1
}
```